paasta_tools/setup_chronos_job.py

```diff
@@ -195,8 +196,9 @@ def main():
             cluster=cluster,
             soa_dir=soa_dir,
         )
-    except NoDeploymentsAvailable:
-        error_msg = "No deployments found for %s in cluster %s" % (args.service_instance, cluster)
+    except (NoDeploymentsAvailable, NoDockerImageError):
+        error_msg = "No deployment found for %s in cluster %s. Has Jenkins run for it?" % (
```

Comment:
I think so. If you are deploying the service for the first time, or just deploying a new instance for the same time, The outcome is pretty much the same and the resolution is the same.